### ISSUE TITLE
`PerspectiveWidget` doesn't handle empty DataFrames

### ISSUE THREAD

**Issue by dhirschfeld** [finos/perspective#2771]
`perspective-python==3.1.0`

### Steps to Reproduce:
```python
>>> from perspective.widget import PerspectiveWidget
>>> PerspectiveWidget([])
# <-- Works fine. A widget is displayed --> #

>>> PerspectiveWidget(pd.DataFrame([]))
---------------------------------------------------------------------------
PyPerspectiveError                        Traceback (most recent call last)
Cell In[6], line 1
----> 1 PerspectiveWidget(pd.DataFrame([]))

File /opt/python/envs/dev310/.pixi/envs/default/lib/python3.10/site-packages/perspective/widget/__init__.py:155, in PerspectiveWidget.__init__(self, data, index, limit, binding_mode, **kwargs)
    152 if limit is not None:
    153     self._options.update({"limit": limit})
--> 155 self.load(data, **self._options)

File /opt/python/envs/dev310/.pixi/envs/default/lib/python3.10/site-packages/perspective/widget/__init__.py:160, in PerspectiveWidget.load(self, data, **options)
    158 """Load the widget with data."""
    159 # Viewer will ignore **options if `data` is a Table or View.
--> 160 super(PerspectiveWidget, self).load(data, **options)

File /opt/python/envs/dev310/.pixi/envs/default/lib/python3.10/site-packages/perspective/widget/viewer/viewer.py:199, in PerspectiveViewer.load(self, data, **options)
    195     raise TypeError(
    196         "Views cannot be loaded directly, load a table or raw data instead"
    197     )
    198 else:
--> 199     self._table = perspective.table(data, name=name, **options)
    200     self._client = perspective.GLOBAL_CLIENT
    202 # If the user does not set columns to show, synchronize viewer state
    203 # with dataset.

File /opt/python/envs/dev310/.pixi/envs/default/lib/python3.10/site-packages/perspective/__init__.py:39, in table(*args, **kwargs)
     37 @functools.wraps(Client.table)
     38 def table(*args, **kwargs):
---> 39     return GLOBAL_CLIENT.table(*args, **kwargs)

PyPerspectiveError: Abort(): Failed to create Table from RecordBatches: Invalid: Must pass at least one record batch or an explicit Schema
```

### Expected Result:
The same as for an empty list. If columns and dtypes are specified then the widget should display but with no data.

**Comment by brianblakely** [comment]
+1, there is a lot of value in being able to initialize a viewer with a blank table, to populate it later.

**Comment by texodus** [comment]
This was fixed in #2854
